docs/CONTRIBUTING.md

```diff
@@ -163,6 +166,65 @@ $> npm run test:as-sqlite
 
 As stated above, the SQLite-based app is accessible via http://localhost:4004
 
+### run tests agains SCP cf PostgreSQL, hyperledger service
+
+SAP offers a [managed PostgreSQL instance on CF](https://discovery-center.cloud.sap/serviceCatalog/postgresql-hyperscaler-option) now. It is also possible to deploy the sample project included here in cds-pg (`__tests__/__assets__/cap-proj`) to SAP CP and connect it to the cf-based hyperscaler PostgreSQL service.
+
+_Note:_ all of the below will work in local \*nix-like environments only (Linux, macOS, WSL on Windows), not on Windows `cmd` or `powershell`.
+
+First, create an instance `cap-proj-database` of the hyperscaler PostgreSQL service.
```

Comment:
I would suggest that we either provide the cf command or include the PostgreSQL service instance creation in the mta.yaml

Reply:
👍 will do